.github/workflows/nightly-maxtext-build.yaml

```diff
@@ -0,0 +1,8 @@
+name: Nightly MaxText build
+'on': workflow_dispatch
```

Comment:
I'm not sure how strict GH actions is, but technically on.workflow_dispatch should be a dict. It doesn't need any args, see the sandbox for an example: https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/workflows/_sandbox.yaml#L4